spec_parser/utils.py

```diff
@@ -205,6 +205,9 @@ def _extract_metadata(self, mdata_list):
                 # report the error
                 self.logger.error(f"{self.name}: Metadata key '{_key}' already exists")
 
+            if _values == ["none"]:  # for some classes subClass is set to none, skip these
```

Comment:
I'd write it: if value is not "none", then save it to metadata

Reply:
done